arduino-ide-extension/src/browser/dialogs/ide-updater/ide-updater-component.tsx

```diff
@@ -0,0 +1,139 @@
+import { nls } from '@theia/core/lib/common';
+import {} from '@theia/core/shared/react';
```

Comment:
Is this import actually importing something? :sweat_smile:

Reply:
💩